test/integration/ovs/ofctrl_test.go

```diff
@@ -0,0 +1,390 @@
+// Copyright 2019 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ovs
+
+import (
+	"fmt"
+	"net"
+	"sync/atomic"
+	"testing"
+
+	binding "github.com/vmware-tanzu/antrea/pkg/ovs/openflow"
+	ofTestUtils "github.com/vmware-tanzu/antrea/pkg/ovs/openflow/testing"
```

Comment:
```suggestion
	openflowtest"github.com/vmware-tanzu/antrea/pkg/ovs/openflow/testing"
```
Don't use mixedCaps for package name and alias: https://golang.org/doc/effective_go.html#package-names

Reply:
Thanks for giving suggestions, I will update accordingly.